src/sql/parts/common/customInputConverter.ts

```diff
@@ -48,6 +49,16 @@ export function convertEditorInput(input: EditorInput, options: IQueryEditorOpti
 			let queryPlanInput: QueryPlanInput = instantiationService.createInstance(QueryPlanInput, queryPlanXml, 'aaa', undefined);
 			return queryPlanInput;
 		}
+
+		//Notebook
+		uri = getNotebookEditorUri(input);
+		if(uri){
+			let notebookData: string = fs.readFileSync(uri.fsPath);
+			let notebookInputModel = new NotebookInputModel('modelViewId', undefined, undefined);
```

Comment:
2 things here:
- For the ModelViewInput, the modelViewId is always unique. Looks like it's based on concatenating the handle with
- You need the handle to be defined or some of the code might break